### ISSUE TITLE
Devportal Applications page accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8386]
### Describe your problem(s)

1. delete button has no accessible name

![image](https://user-images.githubusercontent.com/20179540/85008943-49794700-b17b-11ea-9a76-27d373d5ba50.png)

2. delete confirmation modal should have aria-alertdialog

![image](https://user-images.githubusercontent.com/20179540/85009062-762d5e80-b17b-11ea-84c0-c3bb343cfb14.png)

### How will you implement it

---
